internal/storage/data.go

```diff
@@ -14,3 +16,25 @@ type SeedIssuer struct {
 type SeedData struct {
 	Issuers []SeedIssuer
 }
+
+// SeedDatabase seeds the database using the given storage config.
+func SeedDatabase(config Config) error {
+	switch config.Type {
+	case "":
+		return ErrorMissingEngineType
+	case EngineTypeCRDB:
+	default:
```

Comment:
shouldn't the seed construct also work with the memory database?

Reply:
This was an oversight on my part. I meant to make the in-memory storage engine seed data on initialization but seems like I didn't actually do that.

This should be fixed now.